app/src/main/java/com/antony/muzei/pixiv/provider/PixivArtWorker.kt

```diff
@@ -334,6 +334,20 @@ class PixivArtWorker(
         return duplicateFound
     }
 
+    private fun hasDesiredPixelSize(
+            width: Int,
+            height: Int,
+            minimumWidth: Int,
+            minimumHeight: Int,
+            aspectRatioSetting: Int
+    ): Boolean =
+            when (aspectRatioSetting) {
```

Comment:
What's the rationale behind the different cases depending on if the user wants portrait, landscape, or any art?

Reply:
Example: user selects that he wants only portrait pictures, then I consider it pointless to check image width, because the higher resolution is in this aspect ration height, not width, so it does not really matter, if picture is below width of a display. Same thing for the landscape only.

This could be made into another checkbox of sort or just remove the aspect ratio.